subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -267,20 +331,32 @@ static bool validate_firmware(u32_t fw_dst_address, u32_t fw_src_address,
 		return false;
 	}
 
-	if (!(((u32_t)fwinfo >= fw_src_address)
-		&& (((u32_t)fwinfo + fwinfo->total_size)
-			< (fw_src_address + fwinfo->size)))) {
+	if ((fwinfo->size > (PM_S0_SIZE))
```

Comment:
Please add a define for this to the dummy values used when compiling bootloader sample standalone.

Reply:
Also are you missing an include of pm_config.h?

Reply:
Done